apng-drawable/src/main/kotlin/com/linecorp/apng/ApngDrawable.kt

```diff
@@ -275,6 +302,14 @@ class ApngDrawable @VisibleForTesting internal constructor(
         }
     }
 
+    private fun isFirstFrame(): Boolean = currentFrameIndex == 0
+
+    private fun isLastFrame(): Boolean = currentFrameIndex == frameCount - 1
+
+    private fun isFirstLoop(): Boolean = currentRepeatCount == 1
+
+    private fun isNotLastLoop(): Boolean = currentRepeatCount < loopCount
```

Comment:
Please change it to `isLastLoop()` because `not` word appears twice like `!isNotLastLoop()`

Reply:
DONE